server/src/domain/domain.constant.ts

```diff
@@ -4,6 +4,7 @@ import { extname } from 'node:path';
 import pkg from 'src/../../package.json';
 
 export const AUDIT_LOG_MAX_DURATION = Duration.fromObject({ days: 100 });
+export const AUDIT_LOG_CLEANUP_DURATION = Duration.fromObject({ days: 101 });
```

Comment:
Why is this needed? Is this an attempt to resolve a potential issue with time differences between the server and the database?

Reply:
yeah. it's simply being cautious. keep stuff for 1 day longer than needed

Reply:
While that sounds reasonable, I'd much rather be accurate. If we were accurately requesting the change log, this would be unnecessary.

Reply:
true. better safe than sorry for now?